sympy/core/basic.py

```diff
@@ -1037,7 +1037,7 @@ def xreplace(self, rule):
         >>> x, y, z = symbols('x y z')
         >>> (1 + x*y).xreplace({x: pi})
         pi*y + 1
-        >>> (1 + x*y).xreplace({x:pi, y:2})
+        >>> (1 + x*y).xreplace({x: pi, y: 2})
```

Comment:
Hi, this is E231. 

http://pep8.readthedocs.org/en/latest/intro.html#error-codes